src/sentry/utils/performance_issues/performance_detection.py

```diff
@@ -944,6 +945,20 @@ def visit_span(self, span: Span) -> None:
         if op not in self.settings.get("allowed_span_ops", []):
             return
 
+        # Ignore anything that looks like an asset
+        data = span.get("data") or {}
+        parsed_url = urlparse(data.get("url") or "")
+        _pathname, extension = os.path.splitext(parsed_url.path)
+        if extension and extension in [".js", ".css"]:
+            return
+
+        # Ignore backendy transactions
+        contexts = span.get("contexts") or {}
+        trace = contexts.get("trace") or {}
+        trace_op = trace.get("op")
+        if trace_op and trace_op not in ["navigation", "pageload", "ui.load", "ui.action"]:
```

Comment:
What about if there is no `trace_op` attached, should we not return in those cases?

Reply:
Not quite yet! I want to discard anything that's _definitely wrong_ (specified as not front-end) rather than anything undefined! Same idea as the extension check above, only discard if definitely incorrect

Reply:
That's fair, good call!

Reply:
Had to take this out anyway, the trace isn't on the span, duh! Am putting in a different mechanism for doing this